### ISSUE TITLE
[Accessibility] Color Contrast is not respected.

### ISSUE THREAD

**Issue by andreacantone** [vanilla-music/vanilla#1065]
- Vanilla Music Version: 1.0.85
- Android Version: 6
- Phone vendor / model: Huawei Mate 7
- Application was downloaded from (use `[x]`):
  - [ ] Play Store
  - [x] F-Droid
  - [ ] Other (Nightly Build)
- Did this work in a previous version?
  - [ ] Yes
  - [ ] No
  - [x] NA / Don't know.

## Description

The contrast ratio between text and background should be at least 4.5 . Some elements don't respect the contrast ratio. 

## Steps to reproduce the issue

1. Activate the app [Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=it&gl=US)
2. Open the app [Vanilla Music](https://play.google.com/store/apps/details?id=ch.blinkenlights.android.vanilla&hl=it&gl=US)
3. Activate the registration from the Accessibility Scanner and use the Vanilla Music app.
4. After using the app, stop the recording.

## Screenshot / Video (If applicable)

![IMG_20201118_155435](https://user-images.githubusercontent.com/25429624/99548534-0ab11c80-29b9-11eb-946b-cffa2c50a949.png)
